requirements-macos-m1.txt

```diff
@@ -1,15 +1,13 @@
 # WARNING: KerasNLP has no official support for MacOS M1 at this time. The
 # following will pull required depenencies from the following external sources.
 # - https://developer.apple.com/metal/tensorflow-plugin/
-# - https://github.com/sun1638650145/Libraries-and-Extensions-for-TensorFlow-for-Apple-Silicon/
 # These are not provided by Google, please review both of these dependencies
 # before proceeding.
 
 # Core deps.
-tensorflow-macos==2.9
-https://github.com/sun1638650145/Libraries-and-Extensions-for-TensorFlow-for-Apple-Silicon/releases/download/v2.9/tensorflow_text-2.9.0-cp39-cp39-macosx_11_0_arm64.whl
-# The metal plugin breaks many tests, so is not enabled by default.
-# tensorflow-metal==0.5.1
+tensorflow-macos~=2.10.0
```

Comment:
I'm not sure we should do this, as this would no longer install tensorflow-text, so it would not be very useful.

Seems like we should either keep what we have here on tf 2.9 (which will allow you to run some stuff on a macbook), provide some instructions for building from source, or ditch this entirely.

I would just leave what we have. I think soon we should be able to provided a better lit support path for macos.

Reply:
Done.  Left MacOS requirements unchanged.